mlrun/model.py

```diff
@@ -377,6 +378,113 @@ def source(self, source):
 
         self._source = source
 
+    def build_config(
```

Comment:
missing docstring

Reply:
This class (`ImageBuilder`) is internal, and nobody is expected to use it directly. The `build_config` method of `KubejobRuntime` is documented, and I'll add a docstring for the project method, but not here.